scripts/.eslintrc

```diff
@@ -5,4 +5,7 @@
   "rules": {
     "no-console": "off",
   },
+  "parserOptions": {
+    "ecmaVersion": 2018
+  }
```

Comment:
This is needed because I'm using some recent JS features in the changelog generation script.